lib/resource-loader.ts

```diff
@@ -20,8 +21,12 @@ export class ResourceLoader implements IResourceLoader {
 		return this.$fs.readJson(this.resolvePath(resourcePath));
 	}
 
-	public get appResourcesDir(): string {
-		return this.resolvePath("App_Resources");
+	public get cordovaAppResourcesDir(): string {
```

Comment:
It's part of the resources, the directory itself is part of the resources and that's why I've placed it here. Placing it inside cordova project would mean it is part of any cordova project, while it is part of our own resources. 
That was my logic, but in case you insist, I'll try moving this to cordova project (not sure what kind of mess will happen).